app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -638,6 +669,22 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
         });
     }
 
+    public void shortDelay() {
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void longDelay() {
+        try {
+            Thread.sleep(500);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
We can include method parameters rather than having two methods doing the same thing except for a different parameter.

Reply:
Sure. I will do that. I also thought for same but rather I made two methods just to classify the range.